lighthouse-core/test/computed/metrics/cumulative-layout-shift-test.js

```diff
@@ -304,7 +304,6 @@ describe('Metrics: CLS', () => {
         const cat = 'loading,rail,devtools.timeline';
         trace.traceEvents.push(
           /* eslint-disable max-len */
-          {name: 'FrameCommittedInBrowser', cat, args: {data: {frame: mainFrame, url: 'https://example.com'}}},
```

Comment:
Noticed these extra test entries when comparing the before/after. They don't hurt anything with our current setup, but [`createTestTrace` creates a mainframe `FrameCommittedInBrowser` event](https://github.com/GoogleChrome/lighthouse/blob/6ba8dc883eb5fb355c92a8f236ee5003b305da37/lighthouse-core/test/create-test-trace.js#L110-L121) already, so these end up doubled up